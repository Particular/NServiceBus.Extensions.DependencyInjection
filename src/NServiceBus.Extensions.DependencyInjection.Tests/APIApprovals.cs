namespace NServiceBus.Extensions.DependencyInjection.Tests
{
    using NUnit.Framework;
    using Particular.Approvals;
    using PublicApiGenerator;

    [TestFixture]
    public class APIApprovals
    {
        [Test]
        public void Approve_NServiceBusExtensionsDependencyInjection()
        {
            var publicApi = typeof(InternalType).Assembly.GeneratePublicApi(new ApiGeneratorOptions
            {
                ExcludeAttributes = new[] { "System.Runtime.Versioning.TargetFrameworkAttribute", "System.Reflection.AssemblyMetadataAttribute" }
            });
            Approver.Verify(publicApi);
        }
    }
}