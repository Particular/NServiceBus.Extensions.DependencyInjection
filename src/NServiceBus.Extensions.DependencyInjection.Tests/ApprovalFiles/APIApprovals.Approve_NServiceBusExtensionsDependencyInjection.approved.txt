namespace NServiceBus
{
    public class static ContainerExtensions
    {
        public static NServiceBus.ContainerSettings<TContainerBuilder> UseContainer<TContainerBuilder>(this NServiceBus.EndpointConfiguration configuration, Microsoft.Extensions.DependencyInjection.IServiceProviderFactory<TContainerBuilder> serviceProviderFactory) { }
    }
    public class ContainerSettings<TContainerBuilder>
    {
        public Microsoft.Extensions.DependencyInjection.IServiceCollection ServiceCollection { get; }
        public void ConfigureContainer(System.Action<TContainerBuilder> containerConfiguration) { }
    }
}